### PR TITLE
Fix header font

### PR DESCRIPTION
**Pull request by chris-aeviator**
Pleas review and merge!
I will comment in a second on setting up the country specific menus.

### CODE REVIEW

File: ckanext/odm_nav/lib/odm_nav_helper.py [hunk 1]
```diff
@@ -40,12 +40,11 @@
 
 def load_country_specific_menu(country, wpUrl):
   log.info('getting menu for %s',country)
-
   # list of menu endpoints
   if country=='':
-    menu_endpoint= 'http://pp.' + wpUrl + '/wp-json/menus/824'
+    menu_endpoint= 'http://' + wpUrl + '/wp-json/menus/824'
```
Comment (chris-aeviator) [inline ckanext/odm_nav/lib/odm_nav_helper.py:45]: @acorbi 
you setup the country specific menu by filling in the menu id of the 'country-specifig-menu' which is available on all country subsites.
you can see all menu id's on pp.opendevelopmentmekong.net/wp-json/menus pp-cambodia.open.../wp-json/menus, ...
please make sure that wp-json plugin is enabled, PP wp had some plugins disabled again after some days including wp-json & wpckan

File: ckanext/odm_nav/templates/snippets/top_topics.html [hunk 2]
```diff
@@ -1,5 +1,5 @@
-{% set wp_url='http://pp.opendevelopmentmekong.net' %}
-{% set wp_domain='opendevelopmentmekong.net' %}
+{% set wp_url='http://192.168.33.10' %}
```
Comment (acorbi) [inline ckanext/odm_nav/templates/snippets/top_topics.html:1]: @chris-aeviator is the **wp_url** variable used?
I need to take a look at this more in detail but as I understand it, **wp_domain** variable needs to be adapted everytime? I suggest reading the var from the config through a helper method and config.get as in https://github.com/OpenDevelopmentMekong/ckanext-odm_migration/blob/master/ckanext/odm_migration/scripts/utils/ckanapi_utils.py#L410

Comment (kathi-aeviator) [inline ckanext/odm_nav/templates/snippets/top_topics.html:1]: ok. yes good idea using the config variant!

**Comment by acorbi** [comment]
merged via https://github.com/OpenDevelopmentMekong/ckanext-odm_nav/commit/a5d9dabd1cb985cd126662b55b90480d0d9e2e4c

**Pull request closed without merging**